docs/openapi.yaml

```diff
@@ -0,0 +1,826 @@
+
+        - Only fonts installed in the Docker image are loaded (see the fonts
+        section)
+
+        - Images only work using a `base64` encoded source, e.g. `<img
+        src="data:image/png;base64, iVBORw0K... />`
+
+        - `background-color` and `color` CSS properties require an additional
+        `-webkit-print-color-adjust: exact` CSS property in order to work.
+      parameters:
+        - in: header
+          name: Gotenberg-Output-Filename
+          description: >-
+            By default, the API generates a UUID filename.
+            However, you may also specify the filename per request,
+            thanks to the Gotenberg-Output-Filename header.
+            Caution! The API adds the file extension automatically; you don't have to set it.
+          schema:
+            type: string
+          required: false
+        - in: header
+          name: Gotenberg-Trace
+          description: >-
+            The trace, or request ID, identifies a request in the logs.
+
+            By default, the API generates a UUID trace for each request.
+            However, you may also specify the trace per request, thanks to the Gotenberg-Trace header.
+          schema:
+            type: string
+          required: false
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              $ref: '#/components/schemas/HTMLConvertRequestBody'
+        description: >-
+          The request body must have an `index.html` file in the `files` array,
+          as well as all the referred resources on the same level as the
+          `index.html` file. The request can also include `header.html` and
+          `footer.html`, given the limitations in the API description above.
+      responses:
+        '200':
+          $ref: '#/components/responses/SuccessfulPDF'
+        '400':
+          description: Bad Request
+
+  /forms/chromium/convert/markdown:
+    post:
+      tags:
+        - chromium
+      summary: Convert a Markdown file to PDF
+      description: >-
+        Send an HTML file called `index.html` as a multipart form request and
+        embed a markdown file into the HTML file using the Golang template
+        function `toHTML`, the API will convert the markdown and embed it into
+        the HTML this way and render the resulting page. Markdown conversion
+        works almost exactly the same way as HTML conversion, therefore refer to
+        the HTML conversion page for all the options you can use when converting
+        Markdown documents as well. You can optionally include `header.html` and
+        `footer.html` files as part of the request as well.
+
+
+        An example `index.html` file can be as follows:
+
+        ```html
+
+        <!doctype html>
+
+        <html lang="en">
+          <head>
+            <meta charset="utf-8">
+            <title>My PDF</title>
+          </head>
+          <body>
+            {{ toHTML .DirPath "file.md" }}
```

Comment:
The method `toHTML` takes only one parameter now. See https://gotenberg.dev/docs/modules/chromium#markdown.

Reply:
this a detail I removed in the last commit, see other comment